main.py

```diff
@@ -113,6 +113,14 @@ def get(self):
 def get_redirect_route(regex_route, handler, defaults=None):
     """Returns a route that redirects /foo/ to /foo.
 
+    Args:
+        regex_route: str. For regex for routing
+        handler: function. Handler function
+        defaults: dict. For defaults it's None
```

Comment:
This still doesn't describe what the `defaults` will do - like here, you should mention that will be passed in the redirect if there is any.

Reply:
done